docs/reference/setup/install/getting-started-tls-docker.asciidoc

```diff
@@ -0,0 +1,171 @@
+[[getting-started-tls-docker]]
+=== Getting started with TLS and Elasticsearch
+
+Starting with version 6.0.0, x-pack licensed for security (gold, platinum or enterprise subscriptions) https://www.elastic.co/guide/en/elasticsearch/reference/6.0/breaking-6.0.0-xes.html[requires SSL/TLS] encryption for the transport networking layer.
+
+This section demonstrates an easy path to get started with SSL/TLS for both http and transport using the elasticsearch-platinum docker image.
+
+For further details, please refer to {xpack-ref}/encrypting-communications.html[Encrypting Communications] and https://www.elastic.co/subscriptions[available subscriptions].
+
+==== Prepare the environment
+
+Inside a new, empty, directory create the following **four files**:
+
+`instances.yml`:
+["source","yaml"]
+instances:
+  - name: es01
+    dns:
+      - es01 <1>
+      - localhost
+    ip:
+      - 127.0.0.1
+  - name: es02
+    dns:
+      - es02
+      - localhost
+    ip:
+      - 127.0.0.1
+
+<1> Allow use of embedded Docker DNS server names.
+
+`.env`:
+[source,yaml]
+
+CERTS_DIR=/usr/share/elasticsearch/config/x-pack/certificates <1>
+ELASTIC_PASSWORD=PleaseChangeMe <2>
+
+<1> The path, inside the docker image, where certificates are expected to be found.
```

Comment:
"Docker" proper noun.

Reply:
Addressed in d258de65303b51ed3e19539521fc8a69bff6badf